### ISSUE TITLE
when fine-tune the pretrained model from kinetics on ucf101 or hmdb51, the accuracy is lower than it in your paper about 7, 8 percent?

### ISSUE THREAD

**Issue by huangwei1995** [kenshohara/3D-ResNets-PyTorch#117]

**Comment by huangwei1995** [comment]
The pretrained model is downloaded from yours, like  'resnet-18-kinetics.pth',  'resnext-101-kinetics.pth'. I set the scales as [1.0,  0.841, 0,7,7, 0,595, 0.5], the cropping methods as 'corner', sample duration as 16, batch size as 64

**Comment by huangwei1995** [comment]
the learning rate is from 0.001 to 0.000001

**Comment by Cong222** [comment]
i met the same problem with you

**Comment by slighting666** [comment]
@huangwei1995 @Cong222 
Hi, I got the accuracy of val.log, but I want to calculate the accuracy of video level, where can I get the val.json file and how to use eval_ucf101.py?

**Comment by YTHmamba** [comment]
> 

@huangwei1995 hello ,Have you solved it? How to obtain the accuracy in the paper
